account-kit/react/src/components/auth/card/loading/oauth.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect } from "react";
+import { useSignerStatus } from "../../../../hooks/useSignerStatus.js";
+import { useAuthContext, type AuthStep } from "../../context.js";
+import { ContinueWithOAuth } from "../../../../icons/oauth.js";
+import { ConnectionError } from "../error/connection-error.js";
+import { capitalize } from "../../../../utils.js";
+
+interface CompletingOAuthProps {
+  authStep: Extract<AuthStep, { type: "oauth_completing" }>;
+}
+
+export const CompletingOAuth = ({ authStep }: CompletingOAuthProps) => {
+  const { isConnected } = useSignerStatus();
+  const { setAuthStep } = useAuthContext();
+
+  useEffect(() => {
+    if (isConnected) {
+      setAuthStep({ type: "complete" });
+    }
+  }, [isConnected, setAuthStep]);
+
+  if (authStep.error) {
+    return (
+      <ConnectionError
+        connectionType="oauth"
+        oauthProvider={authStep.provider}
+        handleUseAnotherMethod={() => setAuthStep({ type: "complete" })}
```

Comment:
I think you need to put `handleTryAgain` prop here. It should probably be gotten from

```
const { authenticate }  = useOAuthVerify();
...
<ConnectionError
  connectionType="oauth"
  oauthProvider={authStep.provider}
  handleUseAnotherMethod={() => setAuthStep({ type: "initial" })}
  handleTryAgain={authenticate}
  />
```